components/containerPageHeader/styles.js

```diff
@@ -22,17 +22,23 @@
 import { createStyles } from "@solid/lit-prism-patterns";
 
 const styles = (theme) =>
-  createStyles(theme, [], {
+  createStyles(theme, ["pageHeader"], {
     "files-container-header": {
       [theme.breakpoints.up("sm")]: {
         display: "flex",
         flexDirection: "row",
         justifyContent: "space-between",
         alignItems: "center",
+        paddingRight: 0,
+        borderBottom: `1px solid ${theme.palette.grey.A100}`,
       },
       display: "flex",
       flexDirection: "column",
       alignItems: "flex-start",
+      paddingLeft: 0,
+      "& h1": {
+        paddingLeft: "24px",
```

Comment:
Use the theme enabled spacing instead, e.g. `theme.spacing(2.4)` (I think default spacing is 10).